### ISSUE TITLE
On IE - Nav displays differently

### ISSUE THREAD

**Issue by AblazeRich** [mrjasonweaver/flexnav#17]
Hi Jason Weaver, Thanks for this cool bit of code, mostly going ok in my implementation, but I noticed that in IE I don't get the Nav changing at higher resolutions, it just stretches out the design for smaller resolutions. So it still has the Menu drop down, rather than jumping to the next level of Links like in other browsers.
Is there a way around this?
Here's the link to my template site:
http://www.mason-uk.co.uk/responsive/html5boilerplate.html
Thanks
Rich

**Comment by mrjasonweaver** [comment]
I haven't added support for older IE yet. Coming in the next few iterations.

**Comment by AblazeRich** [comment]
Thanks. Any idea when that will be? Another question, do the updates you put up automatically get added into the downloadable code? Or do you have to go through and add them manually?

**Comment by mrjasonweaver** [comment]
I'd say by the weekend and yes, the updates will be available via the "Zip" download button.

**Comment by AblazeRich** [comment]
Hey, Just downloaded the ZIP this morning and I'm still getting issue's in IE. Did I download the right file? Or are you still working on this yourself?
Regards
Rich

**Comment by AblazeRich** [comment]
So any news on this thread Jason? Sorry to be a pest, I got the other responses to the other thread in my inbox and forgot that I had posted in it.... so assumed they were in response to this one.

**Comment by wintnerdzn** [comment]
Just saw my implementation on Windows XP, ugh! There it is, as  AblazeRich states, the menu for small screens across the maximized screen and it doesn't close on clicking a menu item. Any ETA on the fix?

**Comment by mrjasonweaver** [comment]
This plugin, as of now, doesn't have support for old IE. It's a proof-of-concept that will work well with other plugins/polyfills. Take a look at https://github.com/scottjehl/Respond to add media query support for old IE. That's what I use. Then, depending on the design of the site, make css adjustments for IE7/8.

**Comment by wintnerdzn** [comment]
Thanks Jason, I was hoping to not have to add more scripts. Do you know if respond.js will solve the menu not closing problem?

**Comment by mrjasonweaver** [comment]
@AblazeRich @wintnerdzn Just updated this plugin to support IE7/8. Let me know of any issues you may find. Closing this ticket out.

**Comment by wintnerdzn** [comment]
Thanks Jason. Testing (just locally) the nav doesn't close anymore on click/touch. What do I need to do to have the menu close?

**Comment by mrjasonweaver** [comment]
@wintnerdzn Does it work at http://jasonweaver.name/lab/flexiblenavigation/ ?

Also, would be helpful to know the environment you are testing. os/os version and browser/browser version

Thanks.

**Comment by wintnerdzn** [comment]
Just checking this out, I see you've changed a great deal (class names, etc.) since I launched my site.

I have to figure out what's different and how to revise without starting from scratch. i.e. I don't need the "navicon"

Leslie Wintner
www.wintnerdesign.com

**Comment by AblazeRich** [comment]
Hi Jason, I've just tried out the new code in IE 8 (on a Windows XP OS) and although it appears a little better at the larger resolutions, there is now issues at smaller ones.
The issues are that the buttons just keep squashing together at the lower res's. and also that the dropdowns beneath them are appearing strangely at all resolutions.

**Comment by mrjasonweaver** [comment]
Correct. IE 7 and 8 support is for large screens only since those browsers don't understand media queries.

**Comment by gajrose23** [comment]
Hello indyplanets

Thank you for this great technique.  However I am also wondering about the support for IE and am a little confused.  The website says its supports 7 and up, and I have tried to follow a few of the post relating to this, but am still a little lost.  It looks great in the webkit browsers but in IE8 the dropdowns are all squashed really thin.  Do you have any advice on this?

Thanks

Gary

**Comment by mrjasonweaver** [comment]
@gajrose23 I'm going to write up several examples of how one could work with FlexNav and old IE this evening. Will post here when completed.

**Comment by mrjasonweaver** [comment]
@gajrose23 I've released v0.5. This release has what I believe is full responsive support for IE 7/8 (not tested). Care to test?

**Comment by gajrose23** [comment]
hello, thanks for your swift attention.  

It seems to be much improved on IE8 when the window is minimized, and the small condensed version of the drop down menu seems to operate nicely.  

However in full screen, wide browser window, the menu is still in its collapsed drop down state, instead of showing all the separate buttons, and even then doesn't appear to drop down as it should.

**Comment by mrjasonweaver** [comment]
I reverted back. Will work on IE support this weekend.

**Comment by mrjasonweaver** [comment]
@gajrose23 Pushed bug fixes for IE7/8. v.0.5.2

**Comment by gajrose23** [comment]
Hello

Sorry I have been off on holiday and the job I was working on got put on hold.  I am picking it back up again soon, and just wanted to say thank you for the work you did to perfect this technique.  Once the finalised version of the project is complete, I will let you see it, with the credit in place of course!
